### PR TITLE
Update sidebar label

### PR DESCRIPTION
**Pull request by nghi-ly**
## Description & motivation

Update sidebar label for About set up

### CODE REVIEW

File: website/docs/docs/get-started/getting-started/getting-set-up.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 title: Set up dbt Cloud
```
Comment (runleonarun) [inline website/docs/docs/get-started/getting-started/getting-set-up.md]:
```suggestion
title: About Setting up dbt Cloud
```

File: website/docs/docs/get-started/getting-started/getting-set-up.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 id: set-up-dbt-cloud
```
Comment (runleonarun) [inline website/docs/docs/get-started/getting-started/getting-set-up.md:3]: Should this match the page name? I'm not sure what this does for linking?

**Review by runleonarun — COMMENTED** [review]
@nghi-ly Just a quick question and edit then we can 🚢 it!

File: website/docs/docs/get-started/getting-started/getting-set-up.md (hunk 2 above)
Comment (nghi-ly) [reply website/docs/docs/get-started/getting-started/getting-set-up.md:3]: not always. here's one example: https://github.com/dbt-labs/docs.getdbt.com/blob/next/website/docs/docs/deploy/deployment-overview.md

File: website/docs/docs/get-started/getting-started/getting-set-up.md (hunk 1 above)
Comment (nghi-ly) [inline website/docs/docs/get-started/getting-started/getting-set-up.md]:
```suggestion
title: About setting up dbt Cloud
```

Comment (nghi-ly) [reply website/docs/docs/get-started/getting-started/getting-set-up.md]: gonna fix upper S to lower s

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**